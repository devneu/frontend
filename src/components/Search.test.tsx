import React from 'react';
import renderer from 'react-test-renderer';
import SearchWidget from './Search';

describe('SearchWidget', () => {
  it('Should successfully render', () => {
    const wrapper = renderer.create(<SearchWidget />).toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
