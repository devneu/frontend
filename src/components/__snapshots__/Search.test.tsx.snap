// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchWidget Should successfully render 1`] = `
<div
  className="ant-card ant-card-bordered"
>
  <div
    className="ant-card-head"
    style={Object {}}
  >
    <div
      className="ant-card-head-wrapper"
    >
      <div
        className="ant-card-head-title"
      >
        Search
      </div>
    </div>
  </div>
  <div
    className="ant-card-body"
    style={Object {}}
  >
    <form
      className="ant-form ant-form-inline"
      onReset={[Function]}
      onSubmit={[Function]}
    >
      <div
        className="ant-row ant-form-item"
        style={Object {}}
      >
        <div
          className="ant-col ant-form-item-control"
          style={Object {}}
        >
          <div
            className="ant-form-item-control-input"
          >
            <div
              className="ant-form-item-control-input-content"
            >
              <input
                className="ant-input input-search"
                id="searchInput"
                name="search"
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                placeholder="Enter search term..."
                type="text"
                value=""
              />
            </div>
          </div>
        </div>
      </div>
      <div
        className="ant-row ant-form-item"
        style={Object {}}
      >
        <div
          className="ant-col ant-form-item-control"
          style={Object {}}
        >
          <div
            className="ant-form-item-control-input"
          >
            <div
              className="ant-form-item-control-input-content"
            >
              <button
                className="ant-btn ant-btn-primary button-search"
                onClick={[Function]}
                type="submit"
              >
                <span>
                  Go!
                </span>
              </button>
            </div>
          </div>
        </div>
      </div>
    </form>
  </div>
</div>
`;
